src/main/java/service/CalculatorService.java

```diff
@@ -0,0 +1,44 @@
+package service;
+
+import constants.Operator;
+
+import java.util.List;
+
+public class CalculatorService {
+
+    public int calculate(String calculationFormula) {
+
+        Preprocessor preprocessor = new Preprocessor();
+        List<String> processedValue = preprocessor.process(calculationFormula);
+
+        return calculateStart(processedValue);
+    }
+
+    private int calculateStart(List<String> calculateTargets) {
+
+        int result = Integer.parseInt(calculateTargets.get(0));
+
+        for(int i = 1; i < calculateTargets.size(); i+=2) {
+            int nextNumber = Integer.parseInt(calculateTargets.get(i + 1));
+            result = chooseOperator(calculateTargets.get(i))
+                    .calculate(result, nextNumber);
+        }
+
+        return result;
+    }
+
+    private CalculatorInterface chooseOperator(String calculateTarget) {
+
+        Operator operator = Operator.find(calculateTarget);
+        if(operator == null) {
+            throw new IllegalArgumentException("연산자를 찾을 수 없습니다.");
+        }
```

Comment:
```suggestion
        Optional<Operator> operator = Operator.find(calculateTarget);
```
null을 반환하는 대신 Optional을 사용해봐도 좋을 것 같아요
null 반환이 싫다면 find 내부에서 찾지못했을 때 예외를 던지는 방법도 있겠네요!